### PR TITLE
Item data tff 108

### PR DESCRIPTION
**Pull request by Cafferoq**
### Summary

Added some more data to the student's checked-out item list.

### Checklist

- [x] Did you run `npm test`?
- [x] Did you update/add documentation for new methods or changed functionality?

Finally, is it ready to be reviewed and merged: Why not :green_heart: 

### How to Review

1. Pull
2. npm install to get new package used in api test.
2. Look up student id: 123456
3. Checkout an item.
4. Be in shock and awe of the fact that it tells you the model name and item address
5. Recoil at the bad styling. 
6. Remember that the styling is another task.
7. Approve happily. :)

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-108](https://msoese.atlassian.net/browse/TFF-108)

### CODE REVIEW

File: src/store/student-store.js [hunk 1]
```diff
@@ -34,7 +35,7 @@ store.registerHandler('CLEAR_ALL_DATA', () => {
 
 store.registerHandler('CHECKOUT_SUCCESS', () => {
     student.items = student.items.concat(CartStore.getItems());
-    store.emitChange();
```
Comment (gearedmonkey) [inline src/store/student-store.js]: We should probably keep this emitChange no?

File: src/lib/api-client.js [hunk 2]
```diff
@@ -175,6 +175,13 @@ export function searchStudent(id) {
     });
 }
 
+export function justGetAllModels() {
```
Comment (gearedmonkey) [inline src/lib/api-client.js]: Ewww the controller will fix this issue yea?

Comment (thofmann) [inline src/lib/api-client.js]: Right.

File: src/store/student-store.js (hunk 1 above)
Comment (thofmann) [inline src/store/student-store.js]: Yes.

Comment (Cafferoq) [inline src/store/student-store.js]: I don't think it should be there. But I put it back. It'll all be changed once I refactor some of the API logic into controllers anyway.

**Pull request merged**